go/consensus/tendermint/config/config.go

```diff
@@ -0,0 +1,228 @@
+// Package config implements global configuration options.
+package config
+
+import (
+	"fmt"
+	"time"
+)
+
+// Config is the Tendermint configuration structure.
+type Config struct {
+	// Node is a consensus validator.
+	// This additional option exists because it is currently possible to
+	// run a node that is simultaneously a validator and a compute node.
+	Validator bool `yaml:"validator"`
+
+	// Tendermint listen address.
+	ListenAddress string `yaml:"listen_address"`
+	// Tendermint address advertised to other nodes.
+	ExternalAddress string `yaml:"external_address,omitempty"`
+
+	// Tendermint P2P configuration.
+	P2P P2PConfig `yaml:"p2p,omitempty"`
+
+	// Tendermint nodes for which we act as sentry of the form pubkey@IP:port.
+	SentryUpstreamAddresses []string `yaml:"sentry_upstream_addresses,omitempty"`
+
+	// Minimum gas price for this validator.
+	MinGasPrice uint64 `yaml:"min_gas_price,omitempty"`
+
+	// Transaction submission configuration.
+	Submission SubmissionConfig `yaml:"submission,omitempty"`
+
+	// Height at which to force-shutdown the node (in blocks).
+	HaltHeight uint64 `yaml:"halt_height"`
+	// Average amount of time to delay shutting down the node on upgrade.
+	UpgradeStopDelay time.Duration `yaml:"upgrade_stop_delay"`
+
+	// Tendermint ABCI configuration.
+	ABCI ABCIConfig `yaml:"abci,omitempty"`
+
+	// Consensus state sync configuration.
+	StateSync StateSyncConfig `yaml:"state_sync,omitempty"`
+
+	// Supplementary sanity checks configuration.
+	SupplementarySanity SupplementarySanityConfig `yaml:"supplementary_sanity,omitempty"`
+
+	// Enable tendermint debug logs (very verbose).
+	LogDebug bool `yaml:"log_debug,omitempty"`
```

Comment:
Should this go under the debug options?

Reply:
Hmm, it kind of makes sense to leave it here because it only sets the Tendermint log level to debug and isn't some unsupported debug flag.